common/flags/tests/FlagsTest.java

```diff
@@ -49,7 +50,7 @@ public void defaultsTest() {
     assertEquals(FLAG_SHOULD_HAVE_VALUE, new Integer(123), FlagDescTestClass.integerFlag.get());
     assertEquals(FLAG_SHOULD_HAVE_VALUE, new Long(123456789L), FlagDescTestClass.longFlag.get());
     assertEquals(FLAG_SHOULD_HAVE_VALUE, new Double(1.23), FlagDescTestClass.doubleFlag.get());
-    assertEquals(leftOverArgs.size(), 0);
+    assertEquals(0, leftOverArgs.size());
```

Comment:
Why did you switch the order? The expected value is supposed to be first.

Reply:
I can't speak for @val-fed, but I believe having the constant or _expected value_ first is the idiomatic way to write JUnit 4 `Assert`-based assertions. The JUnit 4 javadocs seem to confirm this: https://junit.org/junit4/javadoc/4.8/org/junit/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)

Reply:
You're right. I just got confused. @val-fed you can ignore this comment :)
Thanks @jbduncan !